### DISCUSSION TITLE
How should we handle complex icons?

### DISCUSSION THREAD

**Discussion by wwwescape** [simple-icons/simple-icons#5975] [Help with SVGs] [answered]
I've been thinking of submitting a few icons that contain shadows, 3D renders, or multiple colors/gradients in the same shape.

How should we handle these icons?

A few examples are:

**Indian Super League** (Issue: #5935)
![image](https://user-images.githubusercontent.com/651507/122156539-ccb0a400-ce86-11eb-852b-f1fbb63fc825.png)

**Webex** (Issue: #5943)
![image](https://user-images.githubusercontent.com/651507/122156637-01246000-ce87-11eb-9bab-964b3cfadf4e.png)

**ICICI Bank** (Issue: #6020)
![image](https://user-images.githubusercontent.com/651507/122156665-139e9980-ce87-11eb-95a7-8b9c3d4c1840.png)

**Comment by adamrusted** [comment, marked as answer]
So typically, I would suggest extracting the icon, resizing it to 24px, and then adding a 0.5pt outline between colour changes, or "height changes". A lot of our Google icons use this style already, and while it's not ideal - it does get the point across in the monochromatic style we go for.
